phpunit.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd"
+         colors="true"
+         verbose="true"
+         bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite name="php-photoslibrary Test Suite">
+            <directory>tests/Unit/V1/</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">src/Google/Photos/Library/V1/</directory>
```

Comment:
This also needs the `/Types` path I believe:
```
<directory suffix=".php">src/Google/Photos/Library/Types/</directory>
```

Reply:
can do- what about the GBPMetadata folder ?

Reply:
We don't have any tests that touch the `GBPMetadata` folder, so I don't think we need to include it here.
However, it does sound like good practice to include to include both packages from the source directory. 
I think we can just shorten these to `.../Photos` - that way the `Types` and `Library.V1` are both included.

Reply:
as with composer.json, I have taken the slightly lazier option of shortening to `src/Google/` in 5bab386